sdk/documentintelligence/ai-document-intelligence-rest/CHANGELOG.md

```diff
@@ -14,5 +14,97 @@
+- **Breaking Changes (with the `@azure/ai-form-recognizer` SDK)** - API shapes have been designed from scratch to support the new Rest Level Client for the Document Intelligence service. Please refer to the [Readme](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/documentintelligence/ai-document-intelligence-rest/README.md) and [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/documentintelligence/ai-document-intelligence-rest/samples) for more understanding.
+
+### `"2023-10-31-preview"` Service API version
+
+The new `"2023-10-31-preview"` service version comes with some new features and a few breaking changes when compared to the API versions supported by the `@azure/ai-form-recognizer` library.
+
+**New Features**
+
+- **Markdown content format**
+
+  Supports output with Markdown content format along with the default plain _text_. For now, this is only supported for "prebuilt-layout". Markdown content format is deemed a more friendly format for LLM consumption in a chat or automation use scenario.
+
+  Service follows the GFM spec ([GitHub Flavored Markdown](https://github.github.com/gfm/)) for the Markdown format. Also introduces a new _contentFormat_ property with value "text" or "markdown" to indicate the result content format.
+
+  ```ts
+  import DocumentIntelligence from "@azure-rest/ai-document-intelligence";
+  const client = DocumentIntelligence(process.env["DOCUMENT_INTELLIGENCE_ENDPOINT"], {
+    key: process.env["DOCUMENT_INTELLIGENCE_API_KEY"],
+  });
+
+  const initialResponse = await client
+    .path("/documentModels/{modelId}:analyze", "prebuilt-layout")
+    .post({
+      contentType: "application/json",
+      body: {
+        urlSource:
+          "https://raw.githubusercontent.com/Azure/azure-sdk-for-js/6704eff082aaaf2d97c1371a28461f512f8d748a/sdk/formrecognizer/ai-form-recognizer/assets/forms/Invoice_1.pdf",
+      },
+      queryParameters: { outputContentFormat: "markdown" }, // <-- new query parameter
+    });
+  ```
+
+  Tables, paragraphs, styles, and formulas are represented accordingly and may have breaking changes with regard to the content that you may have received using the older `@azure/ai-form-recognizer` library, which relies on older service API versions.
+
+- **Query Fields**
+
+  When this feature flag is specified, the service will further extract the values of the fields specified via the queryFields query parameter to supplement any existing fields defined by the model as fallback.
+
+  ```ts
+  await client.path("/documentModels/{modelId}:analyze", "prebuilt-layout").post({
+    contentType: "application/json",
+    body: { urlSource: "..." },
+    queryParameters: { queryFields: ["NumberOfGuests", "StoreNumber"] }, // <-- new query parameter
```

Comment:
I believe we are missing `features: ["queryFields"]` (or equivalent syntax) as an additional query parameter, unless the SDK automatically adds it.

Reply:
Good catch, Paul.
I had a list of tests to add after the release, those would have caught this.. could you point me to the documents as well to test these features?